courses/urls.py

```diff
@@ -10,4 +10,12 @@
 router.register(r"courses", views.CourseViewSet, basename="courses_api")
 router.register(r"course_runs", views.CourseRunViewSet, basename="course_runs_api")
 
-urlpatterns = [url(r"^api/", include(router.urls))]
+urlpatterns = [
+    url(r"^api/", include(router.urls)),
+    url(r"^courses/(?P<pk>[\d]+)/$", views.CourseView.as_view(), name="course-detail"),
+    url(
```

Comment:
While you're touching these lines, could you switch to `path` and `re_path` where appropriate since `url` is legacy?